src/Sdk/NewsroomContacts/types.ts

```diff
@@ -0,0 +1,36 @@
+import { Culture, UploadcareImage } from '../../types';
+
+export interface NewsroomContactsListRequestOptions {
+    /**
+     * Keyword search term
+     */
+    search?: string;
+}
+
+export interface NewsroomContactsSearchRequestOptions {
+    /**
+     * Keyword search term
+     */
+    search?: string;
+    /**
+     * Prezly Query Language expression to filter contacts by.
+     */
+    query?: string;
+}
+
+export interface NewsroomContactCreateRequest {
+    name: string;
+    email?: string | null;
```

Comment:
I thought [our agreements](https://discourse.prezly.io/t/ordering-properties-alphabetically-in-javascript/186/9?u=kamil) are indefinite and Prezly-wide.

It does not have to be done in this PR, I just want us to:
- be consistent
- abide and respect team decisions

Of course we can re-discuss any decision, I just think in this case it would not be a good time investment.

Reply:
Hi @kamilmielnik,

Thank you for your comment.

I really want to reopen this discussion.

I was fine with the frontend team deciding on conventions as long as it's only affecting the stuff *you* are working with. But [as I mentioned before](https://github.com/prezly/prezly/pull/9088/files#r551201217) it's the backend team working mainly on Prezly models interfaces, so I want this to be convenient for us to do our job. Having the props sorted alphabetically just makes it *much more difficult* for me to trace what has to be added or removed as we change the APIs. I really mean it: it is much more difficult.

I find it unfair that my opinion is discarded, while I spend much more time maintaining these files than you do.

Reply:
Great!

Let's have a call about this and sort it out once and for all.
How about next week, after we wrap up the website? (Wednesday?)

Reply:
Scheduled a call for next Wednesday at 11:00 AM (BRU).